tests/workers/rollout/rollout_vllm/test_vllm_spmd.py

```diff
@@ -67,6 +68,7 @@ def are_lists_similar(a, b):
     return percentage_difference <= 15
 
 
+@pytest.mark.skip("https://github.com/vllm-project/vllm/issues/16993")
```

Comment:
Skip this test temporary since it's only works with vllm==0.7.3, our ci image is vllm==0.8.5.